extensions/amp-accordion/1.0/test/test-amp-accordion.js

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import '../../../amp-bind/0.1/amp-bind';
```

Comment:
As discussed, this is more of a integration test.

Reply:
Thanks, I've moved to a separate PR: https://github.com/ampproject/amphtml/pull/30922